docs/sources/view-and-analyze-profile-data/profile-cli.md

```diff
@@ -109,6 +109,9 @@ export PROFILECLI_PASSWORD=<password>
 profilecli <command>
 ```
 
+{{< admonition type="caution" >}}
+If you're querying data from Grafana Cloud Profiles, be sure to use the url of your Grafana Cloud Pyroscope server in `PROFILECLI_URL` (e.g. `https://profiles-prod-001.grafana.net`) and **not** the url of your Grafana Cloud tenant (e.g. `<your tenant>.grafana.net`).
```

Comment:
We don't use Grafana Cloud Pyroscope in any other docs. Do you mean Cloud Profiles or the Pyroscope data source?

Reply:
You're right, I meant to use Cloud Profiles. Fixed in https://github.com/grafana/pyroscope/pull/3526/commits/d41c898149507c24a1d5fe55f3a23435bdb27376